acs5countypoverty.py

```diff
@@ -0,0 +1,47 @@
+from census_response import getCensusResponse
+import json
+
+#Estimate!!Total!!Population for whom poverty status is determined
+pop_table = 'S1701_C01_001E'
+#Estimate!!Below poverty level!!Population for whom poverty status is determined
+poverty_table = 'S1701_C02_001E'
+get_ls = [pop_table, poverty_table]
+
+subject_table = 'https://api.census.gov/data/2018/acs/acs5/subject?'
+geo = 'county:*&in=state:17'
+
+response = getCensusResponse(subject_table, get_ls, geo)
+
+#poverty_data is list of lists, where list elements are rows
+#header row: name, population total, population poverty, fip state, fip county
+poverty_data = response.json()
+
+#Json format
+#{'countyfip': (countyfip = concatenation state and county fip numbers)
+    #{'metric_one': 1234,
+    #'metric_two': 5678}}
+    
+final_json = {}
+
+for d in poverty_data[1:]:
+    #set variables to list elements
+    name, pop_total_str, pop_poverty_str, fip_state, fip_county = d
+    #convert strings to ints
+    pop_total_int = int(pop_total_str)
+    pop_poverty_int = int(pop_poverty_str)
+    #concat strings
+    fip_final = fip_state + fip_county
+    #calculate percent poverty
+    pct_poverty = pop_poverty_int / pop_total_int * 100
+    #create county json
+    county_json = {'name_county':name, 'population_total': pop_total_int, 'population_poverty': pop_poverty_int, 'percent_poverty': pct_poverty}
+    #set county key to county json value
+    final_json[fip_final] = county_json
+
+
+#save file
+with open('final_jsons/acs5countypoverty_output.json', 'w') as f:
```

Comment:
Hey this PR is looking really good! As a last step, could you write a test that does the following? 

1. Runs this script
2. Reads in `final_jsons/acs5countypoverty_output.json` 
3. Asserts that this json has the required format and types ie that it matches
```
#Json format
#{'countyfip': (countyfip = concatenation state and county fip numbers)
    #{'metric_one': 1234,
    #'metric_two': 5678}}
```